modules/persistent-storage-local-install.adoc

```diff
@@ -59,39 +59,35 @@ Once finished, the Local Storage Operator will be listed in the *Installed Opera
 .From the CLI
 . Install the Local Storage Operator from the CLI.
 
-.. Create an object YAML file to define a Namespace, OperatorGroup, and Subscription for the Local Storage Operator,
+.. Create an object YAML file to define an Operator group and subscription for the Local Storage Operator,
 such as `local-storage.yaml`:
 +
 .Example local-storage
 [source,yaml]
 ----
-  apiVersion: v1
-  kind: Namespace
-  metadata:
-    name: local-storage
-  ---
-  apiVersion: operators.coreos.com/v1alpha2
-  kind: OperatorGroup
-  metadata:
-    name: local-operator-group
-    namespace: local-storage
-  spec:
-    targetNamespaces:
-      - local-storage
-  ---
-  apiVersion: operators.coreos.com/v1alpha1
-  kind: Subscription
-  metadata:
-    name: local-storage-operator
-    namespace: local-storage
-  spec:
-    channel: "{product-version}" <1>
-    installPlanApproval: Automatic
-    name: local-storage-operator
-    source: redhat-operators
-    sourceNamespace: openshift-marketplace
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: local-operator-group
+  namespace: local-storage
+spec:
+  targetNamespaces:
+    - local-storage
+---
```

Comment:
In [the preview build](https://bz1827131--ocpdocs.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent-storage-local.html#local-storage-install_persistent-storage-local), this renders in green text and underlined (the same way a `...` would indicating trimmed content).